src/screens/AnonymousChatScreen/index.tsx

```diff
@@ -67,9 +67,9 @@ const AnonymousChatScreen = () => {
     goToChatInfoScreen,
     sendChat,
     selfAnonUserInfo,
-    updateChatContinuity,
     scrollContext,
-    flatListRef: scrollRef
+    flatListRef: scrollRef,
+    updateChatContinuity
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `updateChatContinuity` function has been removed from the destructuring assignment. If this function is not used anymore, it's fine. However, if it's still needed somewhere in the component, this could cause a reference error.
```diff
-    updateChatContinuity,
     scrollContext,
     flatListRef: scrollRef
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->